README.md

```diff
@@ -41,7 +41,10 @@ EmulationStation has a few dependencies. For building, you'll need CMake, SDL2,
 **On Debian/Ubuntu:**
 All of this be easily installed with apt-get:
 ```bash
-sudo apt-get install libsdl2-dev libboost-system-dev libboost-filesystem-dev libboost-date-time-dev libboost-locale-dev libfreeimage-dev libfreetype6-dev libeigen3-dev libcurl4-openssl-dev libasound2-dev libgl1-mesa-dev build-essential cmake fonts-droid
+sudo apt-get install libsdl2-dev libboost-system-dev libboost-filesystem-dev libboost-date-time-dev \
+  libboost-locale-dev libfreeimage-dev libfreetype6-dev libeigen3-dev libcurl4-openssl-dev \
+  libasound2-dev libgl1-mesa-dev build-essential cmake fonts-droid \
+  libvlc5 libvlc-dev libvlccore8 libvlccore-dev
```

Comment:
After installing `vlc-plugin-sdl` video plays fine on the compiled ES.

Please can you update the dependencies / squash and I will merge. thanks.